hack/.golint_failures

```diff
@@ -514,4 +514,5 @@ staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/fischer
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder
 test/e2e/common
+test/e2e/framework/pod
```

Comment:
what is the error from the linter? ideally we should try not to add more exceptions

Reply:
The error is from the name of the config, It wants "PodConfig" to change to "Config" because it is used as pod.PodConfig in other packages

Reply:
Should I rename it Config?

Reply:
sounds fine with me

Reply:
Alright. I will rename it and remove the line from .golint_failures